services/businessconfig/src/main/modeling/swagger.yaml

```
@@ -880,6 +880,11 @@ definitions:
       size:
         description: The size of the field
         type: integer
+      defaultVisibility:
+        description: >- 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)